website/snippets/_visual-editor-prerequisites.md

```diff
@@ -9,7 +9,8 @@ Before using Visual Editor, you should:
     - Snowflake
     - Trino
     - You can access the Visual Editor with adapters not listed, but some features may be missing at this time. 
+- Be using [GitHub](/docs/cloud/git/connect-github) or [GitLab](/docs/cloud/git/connect-gitlab) as your Git provider. 
```

Comment:
Should we also use this prereq snippet in https://docs-getdbt-com-git-ve-git-dbt-labs.vercel.app/guides/visual-editor?step=2?